crates/sui-sdk-types/src/types/effects/v1.rs

```diff
@@ -15,7 +15,7 @@ use crate::types::TransactionEventsDigest;
 pub struct TransactionEffectsV1 {
     /// The status of the execution
     #[cfg_attr(feature = "schemars", schemars(flatten))]
-    status: ExecutionStatus,
+    pub status: ExecutionStatus,
```

Comment:
All fields in V2 are pub while none are pub in V1. We should probably open them up?

Reply:
Reworked this in #60